<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>241</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>225</width>
     <height>95</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>value min</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>value max</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>colormap</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_min">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_max">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_colormap">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Greys</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>gray</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>viridis</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>plasma</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>inferno</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>magma</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>cividis</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Purples</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Blues</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Greens</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Oranges</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Reds</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>YlOrBr</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>YlOrRd</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OrRd</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PuRd</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RdPu</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BuPu</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GnBu</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PuBu</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>YlGnBu</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PuBuGn</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BuGn</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>YlGn</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
